src/browser/styles/bootstrap.grid-only.min.css

```diff
@@ -7,8 +7,1097 @@
+ */ /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
+html {
```

Comment:
hmm, usually I'd expect minified css to be well, minified! Hard to see what was changed in this commit as well because of all the formatting changes...

Reply:
I've remade the change without our on commit hook. It's still hard to see though. What changed is that figure{margin: 1em 40px;}. I wasn't 100% happy with the change anyway so I opted to remove the <Figure> tags instead and leave this css as is.